core/trino-main/src/main/java/io/trino/sql/planner/IrExpressionInterpreter.java

```diff
@@ -883,36 +880,20 @@ protected Object visitRow(Row node, Object context)
         }
 
         @Override
-        protected Object visitSubscript(Subscript node, Object context)
+        protected Object visitSubscript(FieldReference node, Object context)
```

Comment:
shouldn't this method be renamed to `visitFieldDereference` ?

Reply:
Yes, good catch